src/conn/conn_compact.c

```diff
@@ -238,14 +246,18 @@ __background_compact_list_cleanup(
     WT_BACKGROUND_COMPACT_STAT *compact_stat, *temp_compact_stat;
     WT_CONNECTION_IMPL *conn;
     uint64_t cur_time, i;
+    uint64_t time_diff;
 
     conn = S2C(session);
     cur_time = __wt_clock(session);
 
+    WT_UNUSED(time_diff);
```

Comment:
Do we need this one?

Reply:
My bad, that was for debugging.

Reply:
Fixed https://github.com/wiredtiger/wiredtiger/pull/9541/commits/447fd8c07bcd9cdee97983994b076575d231d1f1.